apps/site/lib/site_web/views/schedule_view.ex

```diff
@@ -182,10 +182,10 @@ defmodule SiteWeb.ScheduleView do
       end
 
     pdf_name =
-      if RoutePdf.custom?(pdf) do
-        pdf.link_text_override
-      else
-        [pretty_route_name(route), " schedule"]
+      cond do
+        RoutePdf.custom?(pdf) -> pdf.link_text_override
+        pdf.name != nil -> pdf.name
```

Comment:
Should we check if this isn't an empty string `""`? 🤔

Reply:
Sounds like something that should be handled CMS-side but we can add an explicit check if it makes people feel better :)